getting-started/windows-calico/rancher.md

```diff
@@ -0,0 +1,105 @@
+
+1. Install the Tigera {{site.prodname}} operator and custom resource definitions.
+
+   ```
+   kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
+   ```
+
+1. Download the necessary Installation custom resource.
+
+   ```bash
+   wget {{ "/manifests/custom-resources.yaml" | absolute_url }}
+   ```
+
+1. Update the `calicoNetwork` options, ensuring that the correct pod cidr is set. (Rancher uses `10.42.0.0/16` by default.)
+   Below are example installations for VXLAN and BGP networking using the default Rancher pod cidr:
+
+   **VXLAN**
+
+   ```
+   apiVersion: operator.tigera.io/v1
+   kind: Installation
+   metadata:
+     name: default
+   spec:
+     # Configures Calico networking.
+     calicoNetwork:
+       bgp: Disabled
+       # Note: The ipPools section cannot be modified post-install.
+       ipPools:
+       - blockSize: 26
+         cidr: 10.42.0.0/16
+         encapsulation: VXLAN
+         natOutgoing: Enabled
+         nodeSelector: all()
+   ```
+
+   **BGP**
+
+   ```
+   apiVersion: operator.tigera.io/v1
+   kind: Installation
+   metadata:
+     name: default
+   spec:
+     # Configures Calico networking.
+     calicoNetwork:
+       # Note: The ipPools section cannot be modified post-install.
+       ipPools:
+       - blockSize: 26
+         cidr: 10.42.0.0/16
+         encapsulation: None
+         natOutgoing: Enabled
+         nodeSelector: all()
+   ```
+
+   > **Note**: For more information on configuration options available in this manifest, see [the installation reference]({{site.baseurl}}/reference/installation/api).
+   {: .alert .alert-info}
+
+1. Apply the updated custom resources:
+
+   ```bash
+   kubectl create -f custom-resources.yaml
+   ```
+
+1. [Install and configure calicoctl]({{site.baseurl}}/getting-started/clis/calicoctl/install)
+
+1. [Configure strict affinity]({{site.baseurl}}/getting-started/windows-calico/quickstart#configure-strict-affinity-for-clusters-using-calico-networking)
+
+1. Finally, follow the {{site.prodnameWindows}} [quickstart guide for Kubernetes]({{site.baseurl}}/getting-started/windows-calico/quickstart#install-calico-for-windows)
+
+   > **Note**: For the default Rancher values for the service cidr and DNS cluster IP, see the {% include open-new-window.html text='services config' url='https://rancher.com/docs/rke/latest/en/config-options/add-ons/network-plugins#disabling-deployment-of-a-network-plug-in' %}.
+   {: .alert .alert-info}
+
+1. Check the status of the nodes with `kubectl get nodes`. If you see that the Windows node has the status `Ready`, then you have a {{site.prodnameWindows}} on RKE cluster ready for Linux and Windows workloads!
+
+### Next steps
+
+- [Try the basic policy demo]({{site.baseurl}}/getting-started/windows-calico/demo)
+- [Secure hosts by installing Calico on hosts]({{site.baseurl}}/getting-started/bare-metal/about)
```

Comment:
I thought Calico for Windows did not support non-cluster hosts...

Reply:
It doesn't! Removed the paste error